tools/tu_collector/tu_collector/tu_collector.py

```diff
@@ -281,6 +281,10 @@ def zip_tu_files(zip_file, compilation_database, write_mode='w'):
 
     add_sources_to_zip(zip_file, tu_files)
 
+    with zipfile.ZipFile(zip_file, 'a') as archive:
```

Comment:
Please extend our test suite to check that the zip file contains the `compilation_database.json` file:
https://github.com/Ericsson/codechecker/blob/master/tools/tu_collector/tests/tu_collector_test.py#L53